docs/general/kakarot_components.md

```diff
@@ -9,54 +9,84 @@ The entire Kakarot protocol is composed of 4 different contracts:
 
 ## Kakarot
 
-The main Kakarot contract is located at: `./src/kakarot/kakarot.cairo`.
+The main Kakarot contract is located at:
+[`./src/kakarot/kakarot.cairo`](../../src/kakarot/kakarot.cairo).
 
-This is the core contract which is capable of parsing and executing ethereum bytecode.
+This is the core contract which is capable of executing decoded ethereum
```

Comment:
might be good here to talk about how in our system, account abstraction is responsible for validating signed transactions. maybe we can create an issue for a diagram or something if one doesnt exist.

Reply:
there is actually a stalled issue for a diagram update! see #380 !